language-plutus-core/src/Language/PlutusCore/Type.hs

```diff
@@ -262,7 +262,7 @@ instance PrettyCfg (NameWithType a) where
     prettyCfg cfg@(Configuration _ True) (NameWithType n@(Name (_, ty) _ _)) = parens (prettyCfg cfg n <+> ":" <+> prettyCfg cfg ty)
     prettyCfg cfg@(Configuration _ False) (NameWithType n) = prettyCfg cfg n
 
-newtype NormalizedType tyname a = NormalizedType { getNormalizedType :: Type tyname a }
+newtype NormalizedType tyname a = NormalizedType { unNormalizedType :: Type tyname a }
```

Comment:
I am not sure that this is a good change. At least, as a non-native English speaker, it is very tempting to read this as getting a type that is not normalised (which is the opposite of what it is).

Reply:
It would be `unnormalizedType` then. My preference is to use `un` as a prefix consistently across the code base, because stuff like `getReaderT`, `getCompose`, etc looks silly. Though, not too important.

Reply:
I actually only did this because it was in the Haskell style guide that someone linked earlier that you should always use `get<TYPENAME>` as the unwrapping function. I don't have strong feelings about it.

Reply:
I will rename this.